Stripe/STPPaymentIntentParams.m

```diff
@@ -60,6 +61,19 @@ - (NSString *)description {
     return [NSString stringWithFormat:@"<%@>", [props componentsJoinedByString:@"; "]];
 }
 
+- (NSString *)setupFutureUsageRawString {
+    if (!self.setupFutureUsage) {
+        return nil;
+    }
+    NSDictionary<NSNumber *, NSString *> *map = @{
```

Comment:
For things like this, I prefer having a switch statement with all the enum values. That way we get a compile error if we add a new one and don't update this method